circle.yml

```diff
@@ -11,9 +11,10 @@ machine:
 dependencies:
   pre:
     - sudo pip install docker-compose
+    - sudo cp /usr/bin/docker /usr/local/bin/docker
```

Comment:
Just out of interest, why the cp? Could we not execute Docker in /usr/local/bin if present and fall back to /usr/bin if not? It might be useful to add a comment to explain this step to others looking at the circle.yml in future - especially if Circle changes where Docker is installed.